apps/web/utils/actions/group.ts

```
@@ -12,33 +13,118 @@ import {
+
+        await generateGroupItemsFromPrompt(
+          group.id,
+          user,
+          gmail,
+          token.token,
+          name,
+          prompt,
+        );
+      }
+
       revalidatePath(`/automation`);
```

Biome

[error] Do not use template literals if interpolation and special-character handling are not needed.

Unsafe fix: Replace with string literal


(lint/style/noUnusedTemplateLiteral)